### PR TITLE
[17.0][ADD] product_price_discounted_visibility

### PR DESCRIPTION
**Pull request by Anxo82**

### CODE REVIEW

**Comment by Anxo82** [comment]
@dalonsod Review please.
Thanks

**Comment by dalonsod** [comment]
@lmiguens-solvos please check appearance with most of columns visible:

imagen

Another option:
imagen

And other:
imagen

Which do you prefer?

**Comment by lmiguens-solvos** [comment]
I prefer the last option, because it is a shorter text and the one the client already uses.

**Comment by dalonsod** [comment]
@Anxo82 please update addon adding such legend ("Net") with its translation only in views

**Comment by Anxo82** [comment]
@dalonsod Done! Review please.
Thanks

File: product_price_discounted_visibility/static/description/index.html [hunk 1]
```diff
@@ -0,0 +1,416 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (dalonsod) [inline product_price_discounted_visibility/static/description/index.html]: This line seems to be remove if we want to skip usual Odoo warning during installation

**Review by dalonsod — APPROVED** [review]
:+1: see comment and I'll merge it

**Comment by Anxo82** [comment]
Done! Thanks

**Pull request merged**